packages/kbn-validate-next-docs-cli/README.md

```diff
@@ -0,0 +1,5 @@
+# @kbn/validate-next-docs-cli
+
+Clone a bunch of repos needed to build the next-docs with the latest version of the changes in the repo. This is only used for validating changes in CI with full validation of all IDs. This isn't intended for local development (except for debugging).
```

Comment:
This seems like it could cause Kibana CI to break in a way we can't fix via other repo's docs.  Is there a way to limit this to only build kibana docs?

Reply:
It could, but at this point Kibana is the only repo without a docs block on PRs so we're actually the ones breaking things in other repos. Additionally this would only impact other people working on docs, so I'm not super concerned about it personally.

Reply:
Additionally, no, there isn't any way to validate our docs without building everything. It's a flaw in the current docs system but it's one the docs team is aware of.